sqlx-sqlite/src/any.rs

```diff
@@ -6,7 +6,7 @@ use futures_core::future::BoxFuture;
 use futures_core::stream::BoxStream;
 use futures_util::{StreamExt, TryFutureExt, TryStreamExt};
 
-use sqlx_core::any::{
+pub use sqlx_core::any::{
```

Comment:
Same here, obviously.

Reply:
[rewrite use and address implementation concerns](https://github.com/launchbadge/sqlx/pull/3608/commits/b5c279abedf0a82741b7f1802e17479ca2a64f3b)